sparkbox/apprentices-page-next
# Add 16.0 Apprentices

## Description
This PR adds support for the newest class of apprentices.

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. Visit the PR site.
- [ ] Confirm 16.0 apprentices appear under "Current Apprentices".
- [ ] Confirm 15.0 apprentices appear under "Previous Apprentices".
